akka-docs/rst/java/http/http-model.rst

```diff
@@ -269,8 +269,18 @@ provided to parse (or render to) Strings or byte arrays.
   such as the Client API ``Http.get(sys).outgoingConnection`` or the Host Connection Pool APIs ``Http.get(sys).singleRequest``
   or ``Http.get(sys).superPool``, usually need the same settings, however the ``server`` most likely has a very different set of settings.
 
-URI
-------------------------------
+The URI model
+-------------
 
-Akka HTTP offers the ``akka.http.server.raw-request-uri-header`` flag. When enabled a ``Raw-Request-URI`` header will be added
-to each request. This header will hold the original raw requests URI that was used. For an example check the reference configuration.
\ No newline at end of file
+Akka HTTP offers it's own specialised URI model class which is tuned for both performance and idiomatic usage within
```

Comment:
this should be `its` (it's == "it is"), sorry if I made the typo in my suggestion itself (?)